src/components/ErrorModal/ErrorModal.js

```diff
@@ -106,7 +106,7 @@ function ErrorModal({
     }
   });
 
-  const canBeOverridden = stripes.hasPerm('ui-users.overrideItemBlock')
+  const canBeOverridden = stripes.hasAnyPerm('ui-users.overrideItemBlock,ui-users.override-item-block.execute')
```

Comment:
How i understood we need this fix for Ramsons bugfix. I see several problems that i want to discuss:

1. How i understood we use `stripes.hasAnyPerm('ui-users.overrideItemBlock,ui-users.override-item-block.execute')` instead of `stripes.hasPerm('ui-users.override-item-block.execute')` because we have some problem with `replaces` in https://github.com/folio-org/ui-users/pull/2778/files .  `replaces` not applied or something similar. Hello @zburke , @NikitaSedyx  Could you please clarify do you know how this could have happened or maybe you can suggest something to fix it or advise who we can discuss this problem with?
Hello @Terala-Priyanka Could you please correct me if i am wrong?

2. `hasAnyPerm` not released in Ramsons and Ramsons bugfix. Hello @zburke Could you please clarify `hasAnyPerm` will be released in Ramsons bugfix?

CC: @artem-blazhko

Reply:
@Dmitriy-Litvinenko , IIUC, everything you have said is true. The challenge with renaming is that works with "upgraded" tenants where the original permission has been installed and assigned, but not with "new" tenants where the original permission has never existed. One view is that this is a bug. Another view is that this is a design oversight. Either way, it is something we need to accommodate. 

`stripes.hasAnyPerm()` will be included in Ramsons. Use it wherever Module-A uses a renamed permission from Module-B, or wherever Module-A has an okapi interface dependency like `"foo": "1.0 2.0"` that supplies permission `x` in `1.0` and renames it to `y` in `2.0`.